### ISSUE TITLE
Mouseover with LCircleMarker

### ISSUE THREAD

**Issue by alc32** [KoRiGaN/Vue2Leaflet#358]
### Description

I draw a map with circle markers, which radius is a magnitude. 

This worked in v1.2.3 but with v2.0.2 does not work.

v1.2.3:
When mouseover the municipality, cursor changes to hand pointer cursor and it is showed the value. 
![Working example](https://user-images.githubusercontent.com/23075572/54553743-2b8ac880-49b3-11e9-914b-3a7304ed7f25.png)

v2.0.2:
When mouseover the municipality, nothing happens. If i remove l-circle-marker, mouseover function works.

``` 
<l-map
    :zoom="zoom"
    :center="center"
    v-bind:style="{ 'height': height, 'width': '100%'}"
    :options="mapOptions"
  >
    <l-geo-json
      :geojson="geojson"
      :options="geojsonOptions"
    />
    <div
      v-for="(feature, index) in geojson.features"
      :key="'circle' + feature.properties.COD_INE">
      <l-circle-marker
        :lat-lng="getLatLong(feature.properties.CENTER_LAT, feature.properties.CENTER_LONG)"
        :interactive="false"
        :radius="radius[index]">
      </l-circle-marker>
    </div>
</l-map>

....

geojsonOptions: {
        onEachFeature: (feature, layer) => {
          layer.on({
            mouseover: mouseover.bind(this),
            mouseout: mouseout.bind(this),
            click: clickFeature.bind(this)
          })
        }
      }
```

### Browsers Affected

- [x] Chrome
- [x] Firefox
- [x] Edge
- [x] Safari 9
- [x] Safari 8
- [X] IE 11

### Versions
- Leaflet: v1.4.0
- Vue: v2.6.8
- Vue2Leaflet: v2.0.2

**Comment by DonNicoJs** [comment]
Ehi could you provide a code pen / code Sand box with a minimal reproduction repo? So that I can take a look!

**Comment by alc32** [comment]
In addition, RAM dont stop increasing when its re-rendered the map (in the example, just clicking in menu items)

Live example:
https://vueleaflet.firebaseapp.com/

Code:
https://gitlab.com/alc32/vue2-leaflet-issue

Thanks!

**Comment by DonNicoJs** [comment]
@alc32 Thanks I am investigating this

**Comment by alc32** [comment]
mouseover works with "preferCanvas": false, but fails with "preferCanvas": true @DonNicoJs

**Comment by alc32** [comment]
solved with v2.2.1, thanks
